backend/research_groups/views.py

```diff
@@ -23,11 +37,41 @@ def create(self, request, *args, **kwargs):
         # nowy ale później można tu zaimplementować rozsyłanie maili
         members = request.data["members"]
 
+        if not request.user.email in members:
+            members.append(request.user.email)
+
         for member in members:
             if not User.objects.filter(email=member).exists():
                 User.objects.create_user(
                     username=member.split("@")[0],
                     password=member.split("@")[0],
                     email=member,
                 )
```

Comment:
ogólnie chyba lepsze nazewnictwo by się przydało, coś w stylu member_email, bo ja bym zakładał że jeśli jest member to  to może jest instancja usera